scivision/io/reader.py

```diff
@@ -10,8 +10,33 @@
 SCIVISION_YAML_CONFIG = ".scivision-config.yaml"
 
 
-def _parse_config(path: os.PathLike) -> dict:
+def _is_url(path: os.PathLike) -> bool:
+    return urlparse(path).scheme in ("http", "https",)
+
+
+def _parse_url(path: os.PathLike, branch: str = "main"):
```

Comment:
what about if the domain is gitlab or bitbucket?

Reply:
then we can expand the logic here!

Reply:
cool, let's keep github... then REG can help to expand the logic for other relevant domains, hopefully